bundles/org.openhab.binding.unifi/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -491,4 +512,25 @@
 		</state>
 	</channel-type>
 
+	<channel-type id="purpose">
+		<item-type>String</item-type>
+		<label>Purpose</label>
+		<description>Purpose of the network</description>
+		<state readOnly="true">
+			<options>
+				<option value="corporate">Corporate</option>
+				<option value="guest">Guest</option>
+				<option value="remote-user-vpn">VPN</option>
+				<option value="wan">WAN</option>
+				<option value="vlan-only">VLAN Only</option>
```

Comment:
Can you also regenerate the I18N properties file after adding this option?

Reply:
:man_facepalming:  Did that now